src/i18n/content/jp/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy.mdx

```diff
@@ -26,190 +26,7 @@ iOS (以前の XCFramework)、Android、および React Native のモバイル
       </th>
     </tr>
   </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        [v7.4.6 (最新の iOS エージェント リリース)](/docs/release-notes/mobile-release-notes/ios-release-notes/ios-agent-746/)
-      </td>
-
-      <td>
-        2023 年 7 月 24 日
-      </td>
-
-      <td>
-        2024 年 7 月 24 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.5
-      </td>
-
-      <td>
-        2023 年 5 月 31 日
-      </td>
-
-      <td>
-        2024 年 5 月 31 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.4
-      </td>
-
-      <td>
-        2023 年 4 月 17 日
-      </td>
-
-      <td>
-        2024 年 4 月 17 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.3
-      </td>
-
-      <td>
-        2023年3月1日
-      </td>
-
-      <td>
-        2024 年 3 月 1 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.2
-      </td>
-
-      <td>
-        2023 年 2 月 23 日
-      </td>
-
-      <td>
-        2024 年 2 月 23 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.1
-      </td>
-
-      <td>
-        2023 年 2 月 22 日
-      </td>
-
-      <td>
-        2024 年 2 月 22 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.0
-      </td>
-
-      <td>
-        2022 年 10 月 20 日
-      </td>
-
-      <td>
-        2023 年 10 月 20 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.8
-      </td>
-
-      <td>
-        2022 年 8 月 24 日
-      </td>
-
-      <td>
-        2023 年 8 月 24 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.7
-      </td>
-
-      <td>
-        2022 年 7 月 13 日
-      </td>
-
-      <td>
-        2024 年 7 月 13 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.6
-      </td>
-
-      <td>
-        2022 年 6 月 1 日
-      </td>
-
-      <td>
-        2024年6月1日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.5
-      </td>
-
-      <td>
-        2022 年 4 月 22 日
-      </td>
-
-      <td>
-        2024 年 4 月 22 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.4
-      </td>
-
-      <td>
-        2021年11月18日
-      </td>
-
-      <td>
-        2023 年 11 月 18 日
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.3
-      </td>
-
-      <td>
-        2021 年 8 月 23 日
-      </td>
-
-      <td>
-        2023 年 8 月 23 日
-      </td>
-    </tr>
-  </tbody>
+  <EolPage agent="ios" />
```

Comment:
each of these are going to need their corresponding locales

Reply:
ah i didn't see that before. just fixed!